deploy/crds/keycloak.org_keycloaks_crd.yaml

```diff
@@ -69,6 +69,7 @@ spec:
                   type: boolean
               type: object
             instances:
+              default: 1
```

Comment:
It seems this approach doesn't work correctly. Otherwise, Travis build should probably report success: https://travis-ci.org/keycloak/keycloak-operator/builds/622755328?utm_source=github_status&utm_medium=notification